### DISCUSSION TITLE
Is it possible to display a timeout count down in the Menu.

### DISCUSSION THREAD

**Discussion by rweisman** [ipxe/ipxe#869] [Q&A] [answered]
I am replacing grub  as my uEFI PXE boot loader.
grub displays a count down of the timer before the default boots.
Is this possible with iPXE?  I spent the last hour with Google and no joy.

Thank you.

**Comment by mcb30** [comment, marked as answer]
Yes: see the examples to do exactly this at https://ipxe.org/cmd/choose

**Reply by rweisman** [reply]
Michael,
I did not see that to the right for the item,  I expected to be like grub and display at the bottom of the menu.
Trying to make this as similar as possible so our production technicians are not confused when I change over.

I appreciate you and the time you have taken to help me.
